docs/tips-and-tricks.md

```diff
@@ -0,0 +1,125 @@
+<p align="center">
+  <h2 align="center">📖 Tips and tricks for writing a userstyle</h2>
+</p>
+
+&nbsp;
+
+### Live reloading
+
+If you want to see your changes in real-time while using an external editor:
+
+1. Head to `chrome://extensions/` or your browser's equivalent.
+2. Locate the Stylus extension and click on the `Details` button.
+3. Enable `Allow access to file URLs`.
+4. Now you can open the `catppuccin.user.css` file in your browser by typing `file:///path/to/catppuccin.user.css` in the address bar.
+5. Stylus will load on that page, and at this point ensure that `live preview` is enabled.
+6. Now you can make changes in your preferred editor and see real-time changes.
+
+> [!TIP]
+> Further reading: [Stylus Wiki (Initial installation and live reload)](https://github.com/openstyles/stylus/wiki/Writing-UserCSS#initial-installation-and-live-reload)
+
+### Hiding sensitive information in screenshots
+
+When taking screenshots of userstyles, you may want to hide sensitive information (such as your username, email, etc.).
+
+You may use the [Flow Circular](https://fonts.google.com/specimen/Flow+Circular) font to redact details by obscuring any text on the page.
+
+Use this snippet at the top of your userstyle to import and use the font:
+
+```less
+@import url('https://fonts.googleapis.com/css2?family=Flow+Circular&display=swap');
+
+* {
+  font-family: 'Flow Circular', cursive;
+}
+```
+
+### Converting images to WebP
+
+The Catppuccin project prefers to use the [WebP image format](https://en.wikipedia.org/wiki/WebP) for the asset preview images. We recommend using Google's [`cwebp`](https://developers.google.com/speed/webp/docs/cwebp) conversion utility to convert images to WebP.
+
+#### Installation
+
+| Method                                | Instructions                                                                           |
+| ------------------------------------- | -------------------------------------------------------------------------------------- |
+| [Homebrew](https://brew.sh/)          | `brew install webp` [[view cask]](https://formulae.brew.sh/formula/webp)               |
+| [MacPorts](https://www.macports.org/) | `sudo port install webp` [[view port]](https://ports.macports.org/port/webp/)          |
+| Binaries                              | See https://storage.googleapis.com/downloads.webmproject.org/releases/webp/index.html. |
+
+#### Usage
+
+> _See [the full documentation on `cwebp`](https://developers.google.com/speed/webp/docs/cwebp) for further reference._
```

Comment:
we could use the github flavoured tips here like we did for the other further reading before.